runtime_config_linux.go

```diff
@@ -182,7 +182,9 @@ type CPU struct {
 
 // Pids for Linux cgroup 'pids' resource management (Linux 4.3)
 type Pids struct {
-	// Maximum number of PIDs. A value < 0 implies "no limit".
+	// Maximum number of PIDs. A value <= 0 implies "no limit".
+	// While a limit of 0 is technically valid, it is precisely identical to
+	// a limit of 1, so there is no distinction.
```

Comment:
Oh, sorry. What I meant was that "<= 0 means no limit *within the spec*, but a limit of 0 is identical to a limit of 1 in the PIDs controller". I guess I'm mixing specification and justification, based on the discussion in opencontainers/runc#58. I'll fix up the wording to only mention the actual specification and only leave the justification in the commit message.